pegasus/sites.v3/hourofcode.com/public/survey/splat.haml

```diff
@@ -31,19 +38,18 @@ rightbar: blank
     %span.form-required-field *
     %div
       %select#event-country.form-control{name: "event_country_s", type: "select", required: true}
-        %option{value: nil, selected: true}
-        %option{value: "United States"} United States
-        -DB[:geography_countries].all().map{|i| i[:name_s]}.sort.each do |country|
+        %option{value: country, selected: true}= country
+        - DB[:geography_countries].all().map{|i| i[:name_s]}.sort.each do |country|
```

Comment:
I'm confused... can you tell me what this line does then? (new line 34 that has "unless country == 'United States'"